src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public FruitTransaction() {
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(String code) {
+        this.operation = Operation.fromCode(code);
+        //return this;

```

Comment:
There is a commented code which is not a good practice. Please remove any commented out code.

Reply:
ok, done